src/utils/extractFunctionParams.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+module.exports = function (str) {
+  // Regex to match function calls and their parameters
+  const functionCallRegex = /(\w+)\s*\(([^()]*)\)/g;
+  return str.matchAll(functionCallRegex);
```

Comment:
Not sure if this utility is doing the exact same thing, but could we use the existing `parseFunctionArguments` utility?

https://github.com/stylelint-scss/stylelint-scss/blob/master/src/utils/__tests__/parseFunctionArguments.js#L320

Reply:
Right! Thanks